stellargraph/core/element_data.py

```diff
@@ -303,6 +305,37 @@ def _numpyise(d, dtype):
     return {k: np.array(v, dtype=dtype) for k, v in d.items()}
 
 
+class FlatAdjacencyList:
```

Comment:
Could we add a brief doc string here? It'd be good in particular to briefly reference the similarities and differences to CSR matrix.